FindNearby/force-app/main/default/lwc/confirmModal/__tests__/confirmModal.test.js

```diff
@@ -0,0 +1,25 @@
+import { createElement } from 'lwc';
```

Comment:
Either add test cases or delete this file since the tests here don't do anything.

Reply:
Jest does not accept modals, couldn't find a way.
I deleted this file.